mail-manager/src/main/scala/it/pagopa/interop/commons/mail/MailData.scala

```diff
@@ -47,6 +47,14 @@ object Mail {
     def make(c: Context)(args: c.Expr[Any]*): c.Expr[InternetAddress] = apply(c)(args: _*)
   }
 
+  /**
+    *
+    * @param text comma separated emails list
+    * @return
+    */
+  def from(text: String): Either[Throwable, List[InternetAddress]] =
```

Comment:
the method name could be misleading. Calling
```
Mail.from(smth)
```
I would expect a `Mail`-something in return.

I don't have info on the purpose of this change, is it to obtain recipients? In this case would `recipientsFrom` work?
Or is it for a different purpose?